qutebrowser/javascript/webelem.js

```diff
@@ -163,6 +173,7 @@ window._qutebrowser.webelem = (function() {
 
     funcs.find_css = (selector, only_visible) => {
```

Comment:
I'm a bit confused why this function is named `find_css`, shouldn't it be `get_elements` or something like that? Or maybe there's another css acronym that makes sense in this context I don't know about....

Reply:
Because it finds elements (like the other `find_` functions) based on a CSS selector. Maybe those should be named `find_by_*` to make things clearer though?